go.mod

```diff
@@ -119,6 +120,8 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v5 v5.4.3
 )
 
+require github.com/signalfx/signalfx-go v1.34.0 // indirect
```

Comment:
please move this a bit below to the big blog with indirect requires?

Reply:
Done, thanks for pointing that out!